hat/assets/js/apps/Iaso/domains/pages/components/UsersSelect.tsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { FunctionComponent, useCallback } from 'react';
+import { FormikProps, FieldInputProps } from 'formik';
+// @ts-ignore
+import { Select } from 'bluesquare-components';
+import { Chip } from '@material-ui/core';
+
+import getDisplayName, { useCurrentUser } from '../../../utils/usersUtils';
+import { useGetProfiles } from '../../users/hooks/useGetProfiles';
+
+type FormValues = {
+    users: string;
+};
+
+type Props = {
+    field?: FieldInputProps<string>;
+    form?: FormikProps<FormValues>;
+    label?: string;
+};
+
+export const UsersSelect: FunctionComponent<Props> = ({
+    field,
+    form,
+    label = '',
+    ...props
+} = {}) => {
+    const value = field?.value || [];
+    const currentUser = useCurrentUser();
+    const { data, isFetching: isFetchingProfiles } = useGetProfiles();
+    const profiles = data ? data.profiles : [];
+    const profilesList = profiles.map(p => ({
+        value: p.user_id,
+        label: getDisplayName(p),
+    }));
+    const handleChange = useCallback(
+        (newValue: string): void => {
+            const fieldValue: string[] | undefined = newValue
+                ? newValue.split(',')
+                : undefined;
+            if (field && form) {
+                // Disable delete of current user chip
+                if (fieldValue?.includes(`${currentUser.user_id}`)) {
+                    form.setFieldValue(field.name, fieldValue);
```

Comment:
This is not super robust: The selection will only show the current user has selected themselves first. Maybe we should preselect the current user:

https://user-images.githubusercontent.com/38907762/211318257-3cf2c9ac-ca79-4c4a-b67f-bc229e91c666.mov

Reply:
yes i forgot the creation my bad !